.travis.yml

```diff
@@ -34,6 +34,9 @@ env:
 script:
   - free -m
 
+  # Hacky af: But to temporarily fix the travis build breaks
+  - if [[ $TRAVIS_PYTHON_VERSION == 3.6 ]]; then alias python=python3; alias pip=pip3; fi
```

Comment:
I would recommend doing `python=python3.6` and `pip=pip3.6`

Reply:
Curious. I'll try that see if the build fails.

Reply:
Updated the builds should work.

Reply:
Updated. The builds should work.